### DISCUSSION TITLE
Extension Compatibility

### DISCUSSION THREAD

**Discussion by Jem20** [anthony1x6000/ROBLOX2016stylus#41] [Q&A] [answered]
Is it possible to get this to work with extensions like RoPro and BTRoblox?

**Comment by anthony1x6000** [comment, marked as answer]
I unfortunately don't use those extensions and don't develop around them. So there may be some compatibility issues.

However, this repo is open to pull requests. So if you make any fixes to better compatibility, I will gladly merge those. 
If I do get some free time, I might install those extensions and try to make some fixes of my own.

**Comment by Jem20** [comment]
Alright.
